### DISCUSSION TITLE
add different timezone

### DISCUSSION THREAD

**Discussion by 30096ompratap** [agronholm/apscheduler#887] [Q&A]
i have created job but i want to run it for diffenent time zone how can i do it for django-apscheduler and for apscheduler only .
if you can provide doc that would be great

**Comment by agronholm** [comment]
I have no clue how django-apscheduler works. As for APScheduler itself, you can give the alternate timezone as the `timezone` parameter for the trigger, like in the [docs](https://apscheduler.readthedocs.io/en/3.x/modules/triggers/cron.html#module-apscheduler.triggers.cron).
